src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetAuthzIamStep.java

```diff
@@ -36,6 +36,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
         Map<IamRole, String> policyEmails = iamClient.createDatasetResource(userReq, datasetId);
         workingMap.put(DatasetWorkingMapKeys.POLICY_EMAILS, policyEmails);
         return StepResult.getStepResultSuccess();
+
```

Comment:
If you revert this white space change, the the file will have no changes at all

Reply:
I think these should be cleaned up now. Let me know if you still see them.